apps/synapse-oauth-signin/src/mocks/handlers.ts

```diff
@@ -7,7 +7,10 @@ export function resetConsentedInMockService(newValue: boolean) {
   hasConsented = newValue
 }
 
-export const ACCESS_CODE_PROVIDED_BY_SERVER = 'someAccessCode'
+export const ACCESS_CODE_PROVIDED_BY_SERVER = 'some Access Code' //contains value that will be encoded in the redirect to test for double-encoding
```

Comment:
In reality, I don't think the access codes have ever contained values that require encoding (which is why we have never seen the double-encoding bug in the wild), but I'd like to fix it anyway